pcvalidate/pcvalidate.go

```diff
@@ -31,7 +31,7 @@ func init() {
 
 func main() {
 	flag.Usage = func() {
-		fmt.Fprintf(flag.CommandLine.Output(), "Usage: %s [ OPTIONS ] publiccode.yml\n", os.Args[0])
+		fmt.Fprintf(flag.CommandLine.Output(), "UsageXXXX: %s [ OPTIONS ] publiccode.yml\n", os.Args[0])
```

Comment:
I'm not sure I get this :thinking:

Reply:
Oops, printf debuggins strikes again. That shouldn't have been checked in, sorry.